app/src/main/java/de/westnordost/streetcomplete/overlays/mtb_scale/MtbScale.kt

```diff
@@ -0,0 +1,77 @@
+package de.westnordost.streetcomplete.overlays.mtb_scale
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import de.westnordost.streetcomplete.R
+
+import de.westnordost.streetcomplete.view.image_select.GroupableDisplayItem
+import de.westnordost.streetcomplete.view.image_select.Item
+
+enum class MtbScale(
```

Comment:
For consistency, the the title, description, image, ... should be put into extension functions in a separate file, to clearly separate the data from the resources. See e.g. SurfaceItem.kt

Reply:
Don't worry, it was to improve readability and performance on my side, but I'll modify it for you without any worries.